docker/Dockerfile

```diff
@@ -7,7 +7,7 @@ SHELL ["/bin/bash", "-euxo", "pipefail", "-c"]
 
 RUN \
   apt-get update; \
-  apt-get install --no-install-recommends -y git hub openssh-client
```

Comment:
Pls re-generate the image (CONTRIBUTING.md FTW) + bump the version in `docker_executor` in CI config

Reply:
This is technically a breaking change, so bump MAJOR part of the version, semver FTW

Reply:
> Pls re-generate the image (CONTRIBUTING.md FTW) + bump the version in `docker_executor` in CI config

Aww I forgot you did that already IIRC ;D

Reply:
> This is technically a breaking change, so bump MAJOR part of the version, semver FTW

`4.0.0`? I thought it was `3.0.3`

Reply:
No no, I mean the Docker image version (https://hub.docker.com/r/gitmachete/intellij-plugin-ci/tags)

Reply:
So 9.0.0 now

Reply:
`9.0.0` because we are removing support for `hub` right?